dynamics/src/cgParametricMomentum.cpp

```diff
@@ -216,8 +216,12 @@ void CGParametricMomentum<CG>::mEVPStep(const VPParameters& params, const size_t
 
     // Update the stresses according to the mEVP model
 
-    Nextsim::mEVP::StressUpdateHighOrder(
-        params, pmap, smesh, S11, S12, S22, E11, E12, E22, H, A, alpha, beta);
+    std::cerr << "Fatal: Stressupdatehigherorder requires ParametricMomnetumMap with proper "
+                 "template parameter. See headerfile and use Kernel-infrastructure"
+              << std::endl;
+    abort();
+    // Nextsim::mEVP::StressUpdateHighOrder(
+    //     params, pmap, smesh, S11, S12, S22, E11, E12, E22, H, A, alpha, beta);
```

Comment:
Delete commented call, or add TODO/FIXME explaining the commented out code.

Reply:
I don't know why this is commented out. I'm reverting this, but

1. It'll never be reached because of the ``abort()``call
2. If we remove the ``abort()``, then it must be in place for the function to work
3. ``cgParametricMomentum`` is not used anymore, so we should just remove it. I'll make an issue

Reply:
Yes, that didn't work. I'm just adding a comment.